core/src/rpc.rs

```diff
@@ -535,7 +560,7 @@ mod tests {
 
     fn start_rpc_handler_with_tx(
```

Comment:
yikes: getting close to wanting to return a struct here too.  A 🐃 for later...

Reply:
Agreed